app/code/community/Reverb/ReverbSync/Helper/Orders/Creation/Address.php

```diff
@@ -22,8 +26,10 @@ public function addOrderAddressAsShippingAndBillingToQuote(stdClass $reverbOrder
         $shippingAddressObject = $reverbOrderObject->shipping_address;
         if (!is_object($shippingAddressObject))
         {
-            $error_message = $this->__(self::ERROR_NO_ADDRESS);
-            throw new Exception($error_message);
+            // In this event, we still want to create the Magento order. We will populate the address fields with a
+            //      fake placeholder value
+            $this->_addNoAddressToOrderObject($reverbOrderObject);
+            $shippingAddressObject = $reverbOrderObject->shipping_address;
```

Comment:
I can double check this if you'd like but I'm 99.9% sure that Magento will throw an exception if you attempt to create an order with no country code populated on the address object. Unless you want to modify Magento's order address validation (I strongly advise against this) this is the optimal approach to fulfill this need